<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmSelec</class>
 <widget class="QDialog" name="frmSelec">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Seleccionar trabajo</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>951</width>
     <height>231</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Seleccione una fila y pulse aceptar</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="bttAceptar">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bttCancelar">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>270</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selecciones el trabajo que desea consultar:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
